metabase/metabase
# split data permissions selectors

## Changes

- Split `data-permissions` selectors into multiple files
- Convert data permissions selectors to Typescript

## How to verify

- Ensure permissions specs are green
- Ensure the permissions page works the same way as it worked
